qgispluginci/__about__.py

```diff
@@ -47,13 +54,34 @@
 __uri_tracker__ = f"{__uri__}issues/"
 
 # This string might be updated on CI on runtime with a proper semantic version name with X.Y.Z
-__version__ = "__VERSION__"
+__version__ = getenv("GITHUB_REF_NAME", "__VERSION__")
```

Comment:
I don't like the idea of hardcoding something related to Github here, it's always nice to stay independent where we can.
I'd rather keep `__version__ = "__VERSION__"` and the sed replace in the workflow for this file (only this one).

Reply:
What difference does it make since the sed is about getting the ref name in a GitHub CI context?

Personally, I don't like relying on external code (Bash, so POSIX or GNU dependent) to get or expose version number.

Reply:
The sed command is run from the github workflow. It's decoupled. You coud build it from anywhere.
It's also very explicit, you have the info from the workflow.

Otherwise, could we move to something like this 
https://setuptools.pypa.io/en/latest/userguide/pyproject_config.html#dynamic-metadata 

If it's too much, I'm fine like this, it's not that big of a deal, and I'm quite confident it's gonna evolve anyway.